Kernel/ParallelTransport.m

```diff
@@ -245,11 +297,12 @@
 method = OptionValue["Method"];
 param = OptionValue["Parametrization"];
 
-If[param =!= "Mino", Print["Only Mino time parametrization has been implemented for parallel transport."];Return[];];
+If[param =!= "Mino"&& param =!= "Phases", Print["Only Mino time parametrization has been implemented for parallel transport."];Return[];];
```

Comment:
We should change these Print statements to Messages

Reply:
I'll take care of this (and other) changes to ParallelTransport.m

Reply:
Done